spock-core/core.gradle

```diff
@@ -13,6 +13,7 @@ Spock is inspired from JUnit, jMock, RSpec, Groovy, Scala, Vulcans, and other fa
 dependencies {
   compile libs.groovy // easiest way to add Groovy dependency to POM
   compile libs.junit
+  compile libs.annotations
```

Comment:
Not sure what you mean by forcing it on the user (what drawbacks will the user have exactly? It would also eliminate the need for the custom `@Nullable` annotation already available) or how the used annotations could be optional (I don't think that if I put it in comments or put the dependency to runtime (i.e. non-packaged)) would work (however, I did find an update for the annotations on [Maven Central](http://search.maven.org/#artifactdetails%7Corg.jetbrains%7Cannotations%7C13.0%7Cjar))